crates/extensions/c8y_http_proxy/src/actor.rs

```diff
@@ -52,14 +51,12 @@ struct C8YHttpProxyActor {
 
 #[async_trait]
 impl Actor for C8YHttpConfig {
```

Comment:
This `impl Actor for C8YHttpConfig` makes no more sense now. One should have `impl Actor for C8YHttpProxyActor`.

I will push a fix for that.